build_docs.pl

```diff
@@ -555,9 +556,14 @@ sub init_target_repo {
 #===================================
     my ( $repos_dir, $temp_dir, $reference_dir ) = @_;
 
+    my $git_uri = URI->new($Opts->{target_repo});
+    if ( $ENV{GITHUB_USER} ){
+        $git_uri->userinfo( $ENV{GITHUB_USER} . ":" . $ENV{GITHUB_PASS} );
+    }
+
```

Comment:
I think we should probably only try and do this if it *look* like an http uri. The regular docs build uses git-over-ssh and probably should get this decoration. Same for tests that reference the filesystem.

Reply:
Added a check in cccabfdc3ab2f26f094309ce54701a1f1da2ad2b to ensure it's using https before attaching creds to the URI.

Also, as of e3063d6126ab1faab103c778f145f6408e767371, it falls back to the originally-passed argument if the string's scheme is not https.